src/python/pants/backend/python/goals/run_pex_binary.py

```diff
@@ -1,44 +1,28 @@
 # Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-import os
 
 from pants.backend.python.goals.package_pex_binary import PexBinaryFieldSet
 from pants.backend.python.goals.run_helper import (
     _create_python_source_run_dap_request,
     _create_python_source_run_request,
 )
 from pants.backend.python.subsystems.debugpy import DebugPy
-from pants.backend.python.target_types import PexBinaryDefaults, PexLayout
+from pants.backend.python.target_types import PexBinaryDefaults
 from pants.backend.python.util_rules.pex_environment import PexEnvironment
-from pants.core.goals.package import BuiltPackage
 from pants.core.goals.run import RunDebugAdapterRequest, RunFieldSet, RunRequest
 from pants.core.subsystems.debug_adapter import DebugAdapterSubsystem
 from pants.engine.rules import Get, collect_rules, rule
 from pants.engine.unions import UnionRule
-from pants.option.global_options import UseDeprecatedPexBinaryRunSemanticsOption
 from pants.util.logging import LogLevel
 
 
 @rule(level=LogLevel.DEBUG)
 async def create_pex_binary_run_request(
     field_set: PexBinaryFieldSet,
-    use_deprecated_pex_binary_run_semantics: UseDeprecatedPexBinaryRunSemanticsOption,
     pex_binary_defaults: PexBinaryDefaults,
     pex_env: PexEnvironment,
 ) -> RunRequest:
-    if not use_deprecated_pex_binary_run_semantics.val:
```

Comment:
Oh, hm: I think that both of these are inverted.

Reply:
oh nice catch. thanks for correcting it already!